src/interfaces/banner.ts

```diff
@@ -2,7 +2,7 @@ import { AccountOpAction } from 'controllers/actions/actions'
 
 import { Network } from './network'
 
-export type BannerType = 'error' | 'warning' | 'info' | 'success'
+export type BannerType = 'error' | 'warning' | 'info' | 'info2' | 'success'
```

Comment:
I would name it infoSecondary or something like this, but its not necessary to change it